packages/cli/src/commands/deploy.ts

```diff
@@ -28,21 +30,15 @@ export default class Deploy extends Command {
       char: 'e',
       description: 'environment configuration to run',
     }),
-    skipRestoreDependencies: flags.boolean({
-      char: 's',
-      description: 'skips restoring dependencies after deployment',
-    }),
   }
 
   public async run(): Promise<void> {
     const { flags } = this.parse(Deploy)
 
     if (initializeEnvironment(logger, flags.environment)) {
-      await runTasks(
-        flags.skipRestoreDependencies,
-        compileProjectAndLoadConfig({ production: true }),
-        deployToCloudProvider
-      )
+      const deploymentProjectPath = await createProductionSandbox()
+      process.chdir(deploymentProjectPath) // The deployment will work inside the production sandbox
```

Comment:
These two lines are the main change of this PR

Reply:
I tried that! But it didn't work. The reason is that the Booster application configuration expects all files to be in CWD **when we do the  "require" of the project files**, and we can't specify a different CWD in a `require` call. I will try to try again to see what we can do

Reply:
@javiertoledo I finally was able to fixed this. It required more changes than expected, but as a result, now there is much less code that depends on the global `proces.cwd()`. One of the flagrant cases was that the code paths for the lambas were set using `proces.cwd()` which could cause several problems.

Fixed here: f2b535ba39349e5d8006e9d4afefb5009f1cf4c2